libcxx/include/module.modulemap

```diff
@@ -1251,6 +1251,8 @@ module std_private_debug_utils_strict_weak_ordering_check [system] {
   export std_private_type_traits_is_constant_evaluated
 }
 
+module std_private_deque_fwd [system] { header "__fwd/deque.h" }
```

Comment:
Please align `[system]` with `[system]` on line 1256.

Reply:
We align them per subdirectory in the modulemap AFAICT. Why should we change that here?